server/prisma/migrations/20240229214019_purchase_source/migration.sql

```diff
@@ -0,0 +1,9 @@
+-- CreateEnum
+CREATE TYPE "purchase_source" AS ENUM ('online_ticketing', 'admin_ticketing', 'card_reader', 'online_donation');
+
+-- AlterEnum
+ALTER TYPE "state" ADD VALUE 'failed';
+
+-- AlterTable
+ALTER TABLE "orders" ADD COLUMN     "order_source" "purchase_source",
+ADD COLUMN     "order_status" "state" NOT NULL DEFAULT 'in_progress';
```

Comment:
I would like the default value to be none/null if only to ever prevent a pattern of not explicitly setting the state on new orders

Reply:
Done in latest commit.